src/test/java/se/liu/ida/hefquin/queryplan/executable/impl/ops/ExecOpTestBase.java

```diff
@@ -42,6 +37,13 @@
 		public TPFInterface getInterface() { return iface; }
 	}
 
+	protected static class BRTPFServerForTest implements BRTPFServer
+	{
+		final BRTPFInterface iface = new BRTPFInterfaceImpl();
+
+		@Override
+		public BRTPFInterface getInterface() { return iface; }
+	}
```

Comment:
Can you please add an empty line between this method and the next one.

Reply:
Thanks, added (See commit db1ce88).